po/superdesk.pot

```diff
@@ -4697,6 +4753,10 @@ msgstr ""
 msgid "Role"
 msgstr ""
 
+#: scripts/core/lang/missing-translations-strings.js:36
+msgid "role {{role}} is granted new privileges: Please re-login."
```

Comment:
Oh, ok, never mind then. It's not that critical, nor is it that wrong. Should be fine :shipit: